plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -32,6 +32,10 @@ local CatalogCache = Cache:new{
 
 local OPDSBrowser = Menu:extend{
     opds_servers = G_reader_settings:readSetting("opds_servers", {
+        {
+            title = "Standard Ebooks",
```

Comment:
I think I'd prefer to keep Gutenberg at the top, with this one second. Didn't know they had OPDS, thanks! :-)

Reply:
You did actually, https://github.com/koreader/koreader/pull/8441.

Reply:
Heh, well, clearly I didn't try out.
 that specific feed. :-)